sam-hiberry-app/src/orders/utils/doorman.py

```diff
@@ -0,0 +1,75 @@
+# Python's libraries
+import json
+
+# Own's modules
+from errors.util_error import UtilError
+
+# Third-party libraries
+from aws_lambda_powertools import Logger
+
+
+class DoormanUtil(object):
```

Comment:
I get the idea of a doorman , but how about considering a more specific name like 'RequestHandler' or 'ApiRequestHandler'?